pymc3/distributions/discrete.py

```diff
@@ -389,7 +389,7 @@ def _random(self, lower, upper, size=None):
         # as array-like.
         samples = stats.uniform.rvs(lower, upper - lower - np.finfo(float).eps,
                                     size=size)
-        return np.floor(samples).astype('int32')
+        return np.floor(samples).astype('int64')
```

Comment:
`dtype='int64'` is the default for the `Discrete` class. Should that be changed?